static/scripts/loggedin.js

```diff
@@ -232,6 +240,41 @@ window.addEventListener('DOMContentLoaded', function() {
     if (!/^((?!chrome).)*safari/i.test(navigator.userAgent)) {
         setupFirebasePush();
     }
+
+    let  feedbackSelector = document.querySelector('#feedbackType');
+    if(feedbackSelector){
+        feedbackSelector.onchange = function(){
+            if(feedbackSelector.value === "problem"){
```

Comment:
Maybe you can use a json config with some  more generic loops to make the code more readable and easier to maintain?

Reply:
i did, please review, i'm not sure if the for loops i made can be improved (forEach doesn't work)